api/test/gateways/Academy-Benefits/FetchRecord.test.js

```diff
@@ -152,16 +152,16 @@ describe('AcademyBenefitsFetchRecord gateway', () => {
   });
 
   it('logs if the API and DB return the same record', async () => {
-    const record = {...randomRecord1};
-    const record1 = {...randomRecord1};
+    const record = {...randomRecord1()};
```

Comment:
Ah, now you're creating a new object by calling a function here that creates and returns a new object, you shouldn't need to also destructure the returned object into a(nother) new object. 🙂

Reply:
(That means you could just use `const record = randomRecord1()` here, and the same for `record1` just below.)

Reply:
I've removed the destructure